client/src/App.js

```diff
@@ -16,13 +18,17 @@ export default class App extends Component {
     const savedList = this.state.savedList;
     savedList.push(movie);
     this.setState({ savedList });
+    console.log(movie.id)
   };
 
   render() {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <div>Replace this Div with your Routes Paul Apivat!!!!</div>
+        <Route exact path="/" component={MovieList} />
+        {/* <Route path="/movies/:id" component={Movie} /> */}
+        <Route path="/movies/:id" render={props => (<Movie addToSavedList={this.addToSavedList} match={props.match}/>)  } />
```

Comment:
Instead of passing props like `match={props.match}` just spread the rest of the props like `{...props}`